genre.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
+
+class Genre
+  attr_reader :name
+  attr_accessor :items
+
+  def initialize(name)
+    @name = name
+    @id = rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre = self
```

Comment:
With that line, that method would be used outside of the Item class context, and also in the Item class, there is already an assignation for that property, causing redundancy in the property assignation. A good idea would be if we limt the usage of the add_item method to be available only in the Item context.
```suggestion
    ```